cmd/skopeo/proxy.go

```diff
@@ -426,10 +430,6 @@ func (h *proxyHandler) GetManifest(args []any) (replyBuf, error) {
 		return ret, err
 	}
 
-	err = h.cacheTargetManifest(imgref)
-	if err != nil {
-		return ret, err
-	}
 	img := imgref.cachedimg
```

Comment:
Applies to all users: This variable often has a single user, or at most two — I think inlining `imgref.[cachedImg]` would be a tiny bit easier to follow.